tests/functional/processes/ingest/test_clacso_process.py

```diff
@@ -0,0 +1,12 @@
+from processes import CLACSOProcess
+from .assert_ingested_records import assert_ingested_records
+from .assert_uploaded_manifests import assert_uploaded_manifests
+
+
+def test_clacso_process():
```

Comment:
Test looks good! Are we able to get it passing?

Reply:
The `assert_ingested_records` works but the `assert_uploaded_manifests` tests doesn't since they're currently is no storePDFManifest method in the CLACSOService.